### ISSUE TITLE
Change Home Page Widget Images

### ISSUE THREAD

**Issue by metaweidner** [Bridge2Hyku/Bridge2Hyku.github.io#1]
@crocken Please change the home page widget images: https://github.com/Bridge2Hyku/Bridge2Hyku.github.io/blob/master/pages/pages-root-folder/index.md

Let's use icons or stock photos that illustrate the different content areas.

**Comment by metaweidner** [comment]
Yes, the github-toolkit image can stay. Please find substitutes for the other two.

**Comment by metaweidner** [comment]
@crocken The Hyku logo makes sense for the Hyku widget.

**Comment by crocken** [comment]
there is no horizontal hyku logo. it'd be quite a bit of blank space if i slotted the logo in there (i spent a lot of time photoshopping to make a decent looking "hyku(logo)" horizontal thing for the header, and it never worked, thats why i landed on the rectangle that exists now)

**Comment by crocken** [comment]
for "migration planning" in your head do you have something thats library oriented, or something more "planning"-y like https://unsplash.com/photos/FoKO4DpXamQ or https://unsplash.com/photos/qYgDrCHV1B0

**Comment by metaweidner** [comment]
I like the map image better: https://unsplash.com/photos/qYgDrCHV1B0

**Comment by metaweidner** [comment]
The Hyku widget image might need some additional graphics design. Something like a road (or a bridge) leading to the Hyku leaf might be cool.

**Comment by crocken** [comment]
@metaweidner   i've added a map pic for planning and a wireframe sketch as the hyku image...   let me know if this work for you.
